Userland/Libraries/LibJS/Tests/builtins/TypedArray/TypedArray.prototype.buffer.js

```diff
@@ -0,0 +1,31 @@
+// Update when more typed arrays get added
+const TYPED_ARRAYS = [
+    Uint8Array,
+    Uint16Array,
+    Uint32Array,
+    Int8Array,
+    Int16Array,
+    Int32Array,
+    Float32Array,
+    Float64Array,
+];
+
+test("basic functionality", () => {
+    TYPED_ARRAYS.forEach(T => {
+        const typedArray = new T([1, 2, 3]);
+        expect(Object.hasOwnProperty(typedArray, "buffer")).toBeFalse();
```

Comment:
As mentioned on Discord earlier, this is incorrect. This checks whether `Object` has an own property `"1,2,3"`. It doesn't, so the test passes nonetheless.